docs/reportdaily.rst

```diff
@@ -48,12 +48,95 @@ Subcommands
 init
 ~~~~
 
-Is required for the first time use of the programm
-User can enter his data directly or he need to answer some questions provided by this command
+The ``init`` subcommand is required for the first time to initialize the program.
 
-.. code:: bash 
+The user is asked to answer some question. After the input is finished, two files are created:
+
+A configuration file with all the user information and a sqlite database to hold all data.
+
+
+.. code-block:: bash 
 
    reportdaily init
+   reportdaily init  --help
+   reportdaily init  --change [--duration DURATION] [--name Name][--team TEAM] [--team-number TEAM_NUMBER] [--year YEAR]
+
+.. option:: --change, -c
+
+Make changes to the existing config/database
+
+Use the option  `--change` in two different  ways:
+
+ 
+1. Ask for user input
+
+   .. code-block:: bash
+            reportdaily init --change
+      
+   The exisiting config is shown to the user.
+
+   The user is asked to select an option of the configs which requires changes and to enter new values.
+
+2. Provide additional options allow to make changes direct via the command line.
+   
+   
+   .. code:: bash
```

Comment:
I think, there is no `.. code::` directive, only [`.. code-block::`](https://www.sphinx-doc.org/en/1.4.9/markup/code.html#directive-code-block). The `:code:` is an inline macro, it *seems* to work, but I would be cautious.

Maybe you need to revert your change, here and on the other spots.